ST_grid.c

```diff
@@ -867,7 +868,7 @@ static IndivType* _copy_head(IndivType *head)
  _free_head(to->IndvHead); //free_head() frees the memory allocated by the head and the memory allocated by each part of the linked list
  }
 
- //*to = *from;
+ *to = *from;
```

Comment:
I'm a little confused by this line. If to and from both point to the same thing, then they both point to the same fields. Therefore shouldn't to->est_count already be the same as from->est_count?

Reply:
Ya, doesn't really seem to make much sense. Also, note this function is commented out and never utilized. I think for the time being we can leave this as it is, but I am going to create a new issue that we can revisit later when we go carefully through the gridded code to identify whether or not this function can be removed.